app/screens/RestaurantDetail/index.jsx

```diff
@@ -0,0 +1,79 @@
+import { React, useEffect, useState } from 'react';
+import {
+  ScrollView, Text, StyleSheet, FlatList,
+} from 'react-native';
+import axios from 'axios';
+import { Card, Rating, AirbnbRating } from 'react-native-elements';
+import {
+  PhoneIcon as PhoneIconOutline, MapPinIcon as MapPinIconOutline, ClockIcon as ClockIconOutline, ArrowSmallRightIcon as ArrowSmallRightIconOutline, TruckIcon as TruckIconOutline, BookmarkIcon as BookmarkIconOutline,
+} from 'react-native-heroicons/outline';
+import ImgCarousel from './ImgCarousel';
+import Reviews from './Reviews';
+
+const styles = StyleSheet.create({
+  icons: {
+    width: 16,
+    heigth: 16,
+    color: '#55d6bf',
+  },
+  bookmarkIcon: {
+    width: 20,
+    heigth: 20,
+  },
+});
+
+function RestaurantDetail({ route, navigation }) {
+  const item = route.params.item;
+  console.log('yelp', item.yelpId);
+  const [detail, setDetail] = useState({});
+
+  useEffect(() => {
+    axios.get(`http://localhost:8080/api/v1/restaurants/${item.yelpId}`)
```

Comment:
See previous PR for comments about this. This should be left as restaurantId. https://github.com/Full-Metal-Alchemist/NativEats/pull/16#discussion_r1229085709